docs/Technologies.md

```diff
@@ -55,7 +55,7 @@ The FarmData2 code and documentation can be edited with any editor.  However, fo
 
 TODO: Write this for VSCodium.
 
-<!-->
+<!--
```

Comment:
I assumed lines 58-76 were meant to be commented out, so I fixed the opening tag.

Reply:
That is correct.  Good catch!